docs/content/commands/npm-init.md

```diff
@@ -18,19 +19,19 @@ npm init [<@scope>/]<name> (same as `npx [<@scope>/]create-<name>`)
 package.
 
 `initializer` in this case is an npm package named `create-<initializer>`,
-which will be installed by [`npx`](https://npm.im/npx), and then have its
+which will be installed by [`npm-exec`](/commands/npm-exec), and then have its
 main bin executed -- presumably creating or updating `package.json` and
 running any other initialization-related operations.
 
-The init command is transformed to a corresponding `npx` operation as
+The init command is transformed to a corresponding `npm exec` operation as
 follows:
 
-* `npm init foo` -> `npx create-foo`
-* `npm init @usr/foo` -> `npx @usr/create-foo`
-* `npm init @usr` -> `npx @usr/create`
+* `npm init foo` -> `npm exec create-foo`
+* `npm init @usr/foo` -> `npm exec @usr/create-foo`
+* `npm init @usr` -> `npm exec @usr/create`
 
 Any additional options will be passed directly to the command, so `npm init
-foo -- --hello` will map to `npx create-foo --hello`.
+foo -- --hello` will map to `npm exec -- create-foo --hello`.
```

Comment:
might be nice to have the example also include a single-dashed, and an un-dashed, argument, both before and after the lone double dash, to demonstrate how the forwarding works?

Reply:
I gave it a try, it sounds like a good idea in theory but in practice the example gets too long and way more confusing and less helpful than I was hoping for, here's a screenshot of the result for reference:

<img width="771" alt="Screen Shot 2021-04-19 at 1 40 59 PM" src="https://user-images.githubusercontent.com/220900/115279670-ff535000-a114-11eb-8fe3-1637d62cb7bb.png">

Maybe it was just my choice of options? Or maybe seeing the rendered result you also prefer the original shorter example? What do you think? 🤔

Reply:
multiple examples then? basically i'm saying that behavior around `--` is often confusing to developers, and seeing concrete examples can help them (and me, probably, still) understand the semantics.

Reply:
that's a good idea, let me give it a try!

Reply:
I like it ❤️ 

<img width="786" alt="Screen Shot 2021-04-19 at 2 12 00 PM" src="https://user-images.githubusercontent.com/220900/115283368-42afbd80-a119-11eb-9b16-e923c624c996.png">

Reply:
thanks for the suggestions @ljharb!